src/cladetime/cladetime.py

```diff
@@ -312,26 +313,14 @@ def assign_clades(self, sequence_metadata: pl.LazyFrame, output_file: str | None
         else:
             logger.info("Sequence count complete", sequence_count=sequence_count)
 
-        # if there are many sequences in the filtered metadata, warn that clade assignment will
-        # take a long time and require a lot of resources
-        if sequence_count > self._config.clade_assignment_warning_threshold:
-            msg = (
-                f"Sequence count is {sequence_count}: clade assignment will run longer than usual. "
-                "You may want to run clade assignments on smaller subsets of sequences."
-            )
-            warnings.warn(
-                msg,
-                category=CladeTimeSequenceWarning,
-            )
-
```

Comment:
In the main PR comment, you wrote "The bottleneck now appears to be clade assignment itself."  This makes me think it could still be helpful to include this warning?

Reply:
I agree that a warning could be useful, but the wording we had was so ambiguous/arbitrary, e.g., "run longer than usual"

What about keeping somewhat arbitrary warning trigger (which is [in the config](https://github.com/reichlab/cladetime/blob/main/src/cladetime/util/config.py#L26), so easy to change), but changing the warning text to something like:

"About to assign clades to {sequence_count} sequences. The assignment process is resource-intensive, and depending on the limitations of your machine, you may want to use a smaller subset of sequences."

Reply:
That sounds good to me.

Reply:
Added the warning back!